script/playbackstrategy/msestrategy.js

```diff
@@ -65,7 +65,14 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
       }
 
       function onTimeUpdate () {
-        failoverTime = mediaElement.currentTime - timeCorrection;
+        var IN_STREAM_BUFFERING_SECONDS = 20;
+        var dvrInfo = mediaPlayer.getDashMetrics().getCurrentDVRInfo(mediaPlayer.getMetricsFor('video'));
+        failoverTime = mediaElement.currentTime;
```

Comment:
Super picky, but this would be more readable in an `else`. Reading it made me as 'why would `SLIDING` window ever use this?'

But given we plan to redo MSE failover in the near future, happy to approve this for now.